perses/protocols/nonequilibrium_cycling.py

```diff
@@ -523,7 +524,7 @@ def get_estimate(self):
         reverse_work: npt.NDArray[float] = np.array(self.data["reverse_work"])
         free_energy, error = pymbar.bar.BAR(forward_work, reverse_work)
 
-        return free_energy
+        return free_energy * unit.k * self.data['temperature']
```

Comment:
This looks good to get the units, just to be safe, where is the `self.data` coming from here?

Reply:
`self.data` is populated in `NonEquilibriumCyclingProtocol.gather`.

Reply:
We can extract `temperature` from the Protocol's `settings` there and inject it into the data structure used to populate the `ProtocolResult`.